typescript/src/lib/ethereum/artifacts/mainnet/ArbitrumOneL1BitcoinDepositor.json

```diff
@@ -0,0 +1,737 @@
+    "blockHash": "0xca28b9305b8bd5685b90d6dc17819fb68d14198e4812b0675e93aab068836f3f",
+    "transactionHash": "0x51e2686f22fd29a987096d0c27b9d1c7256e64cd37fd0a31a0bc316bbfc2b92b",
+    "logs": [
+      {
+        "transactionIndex": 75,
+        "blockNumber": 20632547,
+        "transactionHash": "0x51e2686f22fd29a987096d0c27b9d1c7256e64cd37fd0a31a0bc316bbfc2b92b",
+        "address": "0x75A6e4A7C8fAa162192FAD6C1F7A6d48992c619A",
+        "topics": [
+          "0xbc7cd75a20ee27fd9adebab32041f755214dbc6bffa90cc0225b39da2e5c2d3b",
+          "0x000000000000000000000000ff79fca71751a5a0c4487a1ace268d6cd2a64db1"
+        ],
+        "data": "0x",
+        "logIndex": 148,
+        "blockHash": "0xca28b9305b8bd5685b90d6dc17819fb68d14198e4812b0675e93aab068836f3f"
+      },
+      {
+        "transactionIndex": 75,
+        "blockNumber": 20632547,
+        "transactionHash": "0x51e2686f22fd29a987096d0c27b9d1c7256e64cd37fd0a31a0bc316bbfc2b92b",
+        "address": "0x75A6e4A7C8fAa162192FAD6C1F7A6d48992c619A",
+        "topics": [
+          "0x8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e0",
+          "0x0000000000000000000000000000000000000000000000000000000000000000",
+          "0x000000000000000000000000949f0adfda95351829e49aeec0f99371a227572d"
+        ],
+        "data": "0x",
+        "logIndex": 149,
+        "blockHash": "0xca28b9305b8bd5685b90d6dc17819fb68d14198e4812b0675e93aab068836f3f"
+      },
+      {
+        "transactionIndex": 75,
+        "blockNumber": 20632547,
+        "transactionHash": "0x51e2686f22fd29a987096d0c27b9d1c7256e64cd37fd0a31a0bc316bbfc2b92b",
+        "address": "0x75A6e4A7C8fAa162192FAD6C1F7A6d48992c619A",
+        "topics": [
+          "0x7f26b83ff96e1f2b6a682f133852f6798a09c465da95921460cefb3847402498"
+        ],
+        "data": "0x0000000000000000000000000000000000000000000000000000000000000001",
+        "logIndex": 150,
+        "blockHash": "0xca28b9305b8bd5685b90d6dc17819fb68d14198e4812b0675e93aab068836f3f"
+      },
+      {
+        "transactionIndex": 75,
+        "blockNumber": 20632547,
+        "transactionHash": "0x51e2686f22fd29a987096d0c27b9d1c7256e64cd37fd0a31a0bc316bbfc2b92b",
+        "address": "0x75A6e4A7C8fAa162192FAD6C1F7A6d48992c619A",
+        "topics": [
+          "0x7e644d79422f17c01e4894b5f4f588d331ebfa28653d42ae832dc59e38c9798f"
+        ],
+        "data": "0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000037169570d846cc05d5848aaa30194d308b355638",
+        "logIndex": 151,
+        "blockHash": "0xca28b9305b8bd5685b90d6dc17819fb68d14198e4812b0675e93aab068836f3f"
+      }
+    ],
+    "blockNumber": 20632547,
+    "cumulativeGasUsed": "6345110",
+    "status": 1,
+    "byzantium": true
+  },
+  "numDeployments": 1,
+  "implementation": "0xFf79fca71751A5A0C4487a1aCE268d6cd2A64Db1",
```

Comment:
Not blocking this PR but I see the implementation contract has one difference compared to `L1BitcoinDepositor` we have on `main`: it additionally extends `Initializable`. What was the reasoning behind it and can we port this back to `main` - in a separate PR - to have those two versions aligned?

Reply:
Definitely not a problem for this PR but I wanted to clarify it and make sure what's deployed is reflected in the codebase. I will leave the discussion open so that we can hear from @lrsaturnino.

Reply:
@pdyraga It's not strictly necessary since OwnableUpgradeable already inherits from Initializable. We added it to make the use of the initializer pattern more explicit and to future-proof the contract against potential changes in the inheritance chain.